app/services/bulk_upload/lettings/year2023/row_parser.rb

```diff
@@ -1201,11 +1210,11 @@ def attributes_for_log
     attributes["benefits"] = field_124
 
     attributes["period"] = field_126
-    attributes["brent"] = field_128
-    attributes["scharge"] = field_129
-    attributes["pscharge"] = field_130
-    attributes["supcharg"] = field_131
-    attributes["tcharge"] = field_132
+    attributes["brent"] = field_128 if all_charges_given?
```

Comment:
Does clearing these at this point mean that any they have provided are not presented to the user as fields that will be cleared on upload?

Reply:
It would show all the charges in the list and the count to be deleted. We're clearing them at this point because if they get past this they're not really invalid, and the charges that were missing would get set to 0 cause we have that as a mechanism to avoid data loss
<img width="575" alt="image" src="https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/54268893/b242d4e3-085a-4e23-848a-9608e747b6db">